app/code/RLTSquare/Training/view/frontend/templates/email.phtml

```diff
@@ -0,0 +1,2 @@
+<h1>Test</h1>
+<h1><?= $block->getViewModel()->sendEmail()?></h1>
```

Comment:
Unnecessary/inappropriate code -  we should not really be using view files (phtml) to execute business logic.